compiler_opt/rl/log_reader.py

```diff
@@ -233,10 +222,10 @@ def read_log(fname: str) -> Generator[Record, None, None]:
     yield from _enumerate_log_from_stream(f, header)
 
 
-def _add_feature(se: tf.train.SequenceExample, spec: TensorSpec,
+def _add_feature(se: tf.train.SequenceExample, spec: tf.TensorSpec,
                  value: TensorValue):
   f = se.feature_lists.feature_list[spec.name].feature.add()
-  if spec.element_type in [ctypes.c_double, ctypes.c_float]:
+  if spec.dtype in [tf.float32, tf.float64]:
     lst = f.float_list.value
   else:
```

Comment:
raise an error if it's not an int format like int8, int32, int64?

Reply:
done. it should never happen, but doesn't hurt either (defensive programming whatnot)